src/modules/todos/graphql/todos.input.ts

```diff
@@ -0,0 +1,47 @@
+import { InputType, Field } from '@nestjs/graphql';
+import { IsBoolean, IsNotEmpty, IsString } from 'class-validator';
+
+@InputType()
+export class CreateTodoInput {
+  @IsString()
```

Comment:
Since we using both for creating and updating Todo. Please create new id to differentiate between the id of project and the id of todo